### ISSUE TITLE
how to show percentage of uploading when using Storage.put to upload file to S3?

### ISSUE THREAD

**Issue by simon998yang** [aws-amplify/amplify-js#1405]
Hi, I just cannot find how.

**Comment by jordanranz** [comment]
Hi @simon998yang, this is actually a feature that we will begin work on this week.

We will update this issue when the feature has been implemented.

**Comment by elorzafe** [comment]
@simon998yang this is already merged and publish #1830 

Here a code snippet on how to use it
```javascript
Storage.put('your/s3/key', 'data', {
  progressCallback(progress) {
    console.log(`Uploaded: ${progress.loaded}/${progress.total}`);
  },
});
```

**Comment by jaronavales** [comment]
How to pass progressCallback property into outside public variable?
